adminimize.php

```diff
@@ -879,22 +879,24 @@ function _mw_adminimize_set_metabox_cp_option() {
 
 		if ( ! isset( $disabled_metaboxes_[ $current_post_type . '_' . $role ][ '0' ] ) ) {
 			$disabled_metaboxes_[ $current_post_type . '_' . $role ][ '0' ] = '';
+
+			continue;
 		}
 
-		$user = wp_get_current_user();
+		$user = $GLOBALS['current_user'];
```

Comment:
Why this change? I think the function returns the current user, the ID so that usage of a globally is not necessary, or?